tests/kani/DynTrait/any_cast_int.rs

```diff
@@ -1,6 +1,10 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
+// kani-flags: --no-restrict-vtable
```

Comment:
Should this changed be postponed to when function pointer restrictions are turned on by default?

Reply:
IMO no: I like that the test suite passes with it enabled and disabled by default, and we have the tracking issue to track this particular issue. While I think we should fix this issue before enabling by default, I actually don't feel super strongly about it, so if we decided to do otherwise, I'd go along with it. I don't think we'd need a test suite failure to remind us?